packages/ecommerce-app-base/src/SkuPicker/SkuPicker.tsx

```diff
@@ -61,7 +69,6 @@ export class SkuPicker extends Component<Props, State> {
     super(props);
     this.setSearchCallback = debounce(() => {
       this.setActivePage(1);
-      this.updateProducts();
```

Comment:
I have removed it because duplicate requests were being issued unnecessarily. If you check the `setActivePage` method, it calls `updateProducts` method itself at the end of its execution, so no need to call if explicitly in the `setSearchCallback`